src/Components/IncomeBlock/IncomeQuestion.js

```diff
@@ -73,11 +74,9 @@ const IncomeQuestion = ({
 
     const menuItems = menuItemKeys.map((menuItemKey, i) => {
       return (
-        <SubHeaderWrapper key={menuItemKey}>
```

Comment:
This `SubHeaderWrapper` broke the onClick functionality of the `menuItems` so I removed it from here as well as the file since it was just a div with a `secondary-heading` css class.

Reply:
Yes, I did.